app/templates/home/chat.hbs

```diff
@@ -1,10 +1,10 @@
 {{#if fastboot.isFastBoot}}
   <div class="main-content">
     <div class="loading-container">
-      <img alt={{t "loading"}} src={{random-loading-spinner}}>
+      <img alt={{t "loading"}} src=<RandomLoadingSpinner />>
```

Comment:
Umm is this one right? I've never seen syntax quite like this. Worth taking a peek at the page and being sure this works.

Reply:
Ah shoot, no this is a helper not a component. The codemod didn't seem to recognize the difference. Nice catch. 🎉